### PR TITLE
Add grid customisation

### PR DESCRIPTION
**Pull request by qsomazzi**

### CODE REVIEW

**Comment by rande** [comment]
Can you also add a class option ? Thanks

**Comment by rande** [comment]
also the key should be named header_style and header_class

**Comment by qsomazzi** [comment]
@rande Done

File: Resources/views/CRUD/base_list_field.html.twig [hunk 1]
```diff
@@ -9,7 +9,7 @@ file that was distributed with this source code.
 
 #}
 
-<td class="sonata-ba-list-field sonata-ba-list-field-{{ field_description.type }}" objectId="{{ admin.id(object) }}">
+<td class="sonata-ba-list-field sonata-ba-list-field-{{ field_description.type }}" objectId="{{ admin.id(object) }}"{% if field_description.options.align is defined %} style="text-align:{{ field_description.options.align }}"{% endif %}>
```
Comment (rande) [inline Resources/views/CRUD/base_list_field.html.twig]: seems an extra commit here

Comment (qsomazzi) [inline Resources/views/CRUD/base_list_field.html.twig]: no, this is the alignement of inner_cells (to override the template default one)

**Comment by qsomazzi** [comment]
@rande done

**Pull request merged**